spec/integration/models/metadata/report_spec.rb

```diff
@@ -0,0 +1,57 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata'
+
+describe GoodData::Report, :constraint => 'slow' do
+  before(:each) do
+    @spec = JSON.parse(File.read("./spec/data/blueprints/test_project_model_spec.json"), :symbolize_names => true)
+    @client = ConnectionHelper.create_default_connection
+    @domain = @client.domain(ConnectionHelper::DEFAULT_DOMAIN)
+    @blueprint = GoodData::Model::ProjectBlueprint.new(@spec)
+    @project = @client.create_project_from_blueprint(@blueprint,
+      token: ConnectionHelper::GD_PROJECT_TOKEN, environment: ProjectHelper::ENVIRONMENT)
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.